src/docfx/config/Config.cs

```diff
@@ -311,6 +311,8 @@ internal class Config : PreloadConfig
         [JsonConverter(typeof(OneOrManyConverter))]
         public string[]? MAMLMonikerPath { get; private set; }
 
+        public JoinTOCConfig[]? JoinTOC { get; private set; }
```

Comment:
It seems `JoinTOC` does not have semantic differences between null and empty array, if so consider
```suggestion
        public JoinTocConfig[] JoinTOC { get; private set; } = Array.Empty<JoinTocConfig>()
```